ReadmeDesignerProjects.md

```diff
@@ -0,0 +1,36 @@
+# WinForms designer
+
+Starting with .NET Core 3.1 it's now possible to use WinForms in sdk-style
+csproj files. However, support for opening files in Designer is still not
```

Comment:
I think this is no longer true. At least that's what we're attempting to prove.

Reply:
Ok, that is true. I'm unaware of designer projects in libpalaso but they can very well be there. I'll leave this file in here, and will update the comment to what we know will be accurate.

Reply:
I think we should delete that file. It's outdated and the content is only partially true nowadays (even with the edit).

Reply:
Ok, I removed the readme.